From: gcarq <gcarq@users.noreply.github.com>
Date: Wed, 13 Aug 2014 19:21:56 +0200
Subject: Update ChatMessage.vala

---
 src/ui/ChatMessage.vala | 15 +++++++++------
 1 file changed, 9 insertions(+), 6 deletions(-)

diff --git a/src/ui/ChatMessage.vala b/src/ui/ChatMessage.vala
index 8cfda90..010d875 100644
--- a/src/ui/ChatMessage.vala
+++ b/src/ui/ChatMessage.vala
@@ -26,20 +26,23 @@ namespace Venom {
     public ChatMessage(IMessage message, bool short_names, bool following, int offset){
       init_widgets();
 
-      if( message.message_direction == MessageDirection.OUTGOING ) {
-        name_label.get_style_context().add_class("own_name");  
-        message_label.get_style_context().add_class("own_message");
-      }
-      
       if( message.is_action ) {
         name_label.set_text( message.get_sender_plain() );
-      } else if(!following) {  
+      } else if(!following) {
+        if( message.message_direction == MessageDirection.OUTGOING ) {
+          name_label.get_style_context().add_class("own_name");
+          message_label.get_style_context().add_class("own_message");
+        }
         if(short_names) {
           name_label.set_text( Tools.shorten_name( message.get_sender_plain() ) );
         } else {
           name_label.set_text( message.get_sender_plain() );
         }
       } else {
+        if( message.message_direction == MessageDirection.OUTGOING ) {
+          name_label.get_style_context().add_class("own_name");
+          message_label.get_style_context().add_class("own_message");
+        }
         name_label.set_text("");
         name_label.set_size_request(offset, -1);
       }
